data/explorations/android_interactions/Test all android interactions.yaml

```diff
@@ -182,6 +182,8 @@ states:
             unicode_str: ''
         rows:
           value: 1
+        catchMisspellings:
```

Comment:
Just to check, in general (if you look across our customization args), are more of them lowerCamelCase or snake_case?